hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestCleanPlanExecutor.java

```diff
@@ -273,7 +273,7 @@ public void testKeepLatestCommits(
    * Test Hudi COW Table Cleaner - Keep the latest file versions policy.
    */
   @ParameterizedTest
-  @ValueSource(booleans = {false, true})
+  @ValueSource(booleans = {false})
```

Comment:
IMO, we at least leave a reason description(or a piece of todo information) for this action so that we could enable it in the future.
It is about the test coverage.
WDYT?

Reply:
Absolutely! 
Sorry, I missed it. Will add in the next revision.